Taboola SDK Swift Sample/Taboola SDK Swift Sample/SDK Standard/UITableView Constraints/TableViewWidgetConstraints.swift

```diff
@@ -0,0 +1,148 @@
+//
+//  UITableViewConstraints.swift
+//  Taboola SDK Swift Sample
+//
+//  Created by Liad Elidan on 24/11/2019.
+//  Copyright © 2019 Taboola LTD. All rights reserved.
+//
+
+import UIKit
+import TaboolaSDK
+
+extension UIColor {
+    static func randomCreation() -> UIColor {
+        func randomColorComponent() -> CGFloat {
+           return CGFloat.random(in: 0...255) / 255
+        }
+        return UIColor(red: randomColorComponent(), green: randomColorComponent(), blue: randomColorComponent(), alpha: 1.0)
+    }
+}
+
+class TableViewWidgetConstraints: UITableViewController, TaboolaViewDelegate {
```

Comment:
Separate the protocols to a different section:

`extension TableViewWidgetConstraints: TaboolaViewDelegate { }
`
Don't implement them within the class.

Reply:
fixed